src/Antiforgery/ref/Microsoft.AspNetCore.Antiforgery.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
     <Compile Include="Microsoft.AspNetCore.Antiforgery.netcoreapp3.0.cs" />
+    <Compile Include="Microsoft.AspNetCore.Antiforgery.Manual.cs" Condition="Exists('Microsoft.AspNetCore.Antiforgery.Manual.cs')" />
```

Comment:
I should add a flag in the src project to controls whether this is added. Otherwise this is too noisy.